packages/cli/src/update-version-notifier.ts

```diff
@@ -35,7 +35,7 @@ const isNewVersionAvailable = (current: string, latest: string) => compare(curre
 const getLatestVersion = async (packageName: string): Promise<string> => {
   const latestUrl = `http://registry.npmjs.org/${packageName}/latest`;
   const response = await fetch(latestUrl);
-  const info = await response.json();
+  const info = await response?.json();
   return info.version;
```

Comment:
If there can be no response, there could be no info, thus info.version will also fail.

Reply:
This is not fail because we handle errors. But this is good catch, Thanks.

I updated this.